packages/shared/utils/port.ts

```diff
@@ -0,0 +1,18 @@
+// copied from vue: https://github.com/vuejs/core/blob/3be4e3cbe34b394096210897c1be8deeb6d748d8/packages/shared/src/general.ts#L90-L112
```

Comment:
```suggestion
import { camelize as camelize_ } from 'vue-demi'

// copied from vue: https://github.com/vuejs/core/blob/3be4e3cbe34b394096210897c1be8deeb6d748d8/packages/shared/src/general.ts#L90-L112
```

Reply:
Wouldn't this result in an error if someone is on a Vue version that doesn't export it?

I don't think it's worth it.

Reply:
`vue-demi` returned an `undefined` on vue 2 when I was testing it

Reply:
Unfortunately not. That would require `vue-demi` to include such exports but it only re-exports everything from vue, meaning that you basically try to import something non-existant from Vue

I created an example for you https://stackblitz.com/edit/vitejs-vite-eikher?file=src%2FApp.vue